Lib/asyncio/streams.py

```diff
@@ -191,7 +191,9 @@ async def _drain_helper(self):
         if not self._paused:
             return
         waiter = self._drain_waiter
-        assert waiter is None or waiter.cancelled()
+        assert waiter is None or waiter.cancelled(), (
+            'Another task is waiting for this stream to drain'
```

Comment:
I think this would be a little clearer if it said, "Another task is already waiting for this stream to drain"

Reply:
I'd make it a proper check too.